tests/unit/CMakeLists.txt

```diff
@@ -51,4 +51,8 @@ if(EXAGO_INSTALL_TESTS)
     DEPENDS
     LOGGING
   )
+
+  if(EXAGO_ENABLE_LOGGING)
+    set_tests_properties(UNIT_TESTS_LOGGER PROPERTIES LABELS "ascent-skip")
+  endif()
```

Comment:
Doesn't look like any tests in this file are skipped on ascent?

Reply:
https://code.ornl.gov/ecpcitest/exasgd/exago/-/jobs/2072690 - see #42

Logging test and OPFLOW tests (#43) are successfully being skipped on Ascent